src/main/java/com/blackduck/integration/jenkins/polaris/service/PolarisCliVersionHandler.java

```diff
@@ -28,14 +28,14 @@ public String extractPolarisCliVersion(InputStream logStream) throws IOException
     }
 
     /**
-     * Compares two Polaris CLI version strings in the format "X.Y.Z" (e.g., "1.0.1").
+     * Compares two Polaris CLI version strings in the format "YYYY.MM.P" (e.g., "2024.9.0").
      *
-     * @param version1 the first version string to compare, in the format "X.Y.Z"
-     * @param version2 the second version string to compare, in the format "X.Y.Z"
+     * @param version1 the first version string to compare, in the format "YYYY.MM.P"
+     * @param version2 the second version string to compare, in the format "YYYY.MM.P"
      * @return an integer that is:
      *         - Negative if version1 is less than version2
      *         - Zero if version1 is equal to version2
-     *         - Positive if version1 is greater than version2
+     *         - Positive if version1 is greater than version2 or major part of the version2 is less than length of 4
```

Comment:
@zaman-akib  Do you mind moving `major part of the version2 is less than length of 4` to a separate line. And, also add, `this is specifically to determine the CLI dev versions, as they are SemVer versions, but not of the format YYYY.MM.P, but of the format X.Y.Z`?